app/screens/LoginScreen.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+import { Image, StyleSheet, Text, View } from "react-native";
+import { Button, TextInput } from "react-native-paper";
+
+import colors from "../config/colors";
+import defaultStyles from "../config/styles";
+import Screen from "../components/Screen";
+
+function LoginScreen({ setIsLoggedIn }) {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  function authenticate() {
+    setIsLoggedIn(true);
+    // if(username === 'admin' && password === 'admin')
+    //   setIsLoggedIn(true);
+  }
+
+  return (
+    <Screen style={styles.container}>
+      <Text style={styles.appName}>KiranaShop</Text>
+      <Image style={styles.logo} source={require("../assets/AppLogo.png")} />
+      <View style={styles.form}>
+        <TextInput
+          style={styles.inputTextBox}
+          mode="outlined"
+          label="Username"
+          value={username}
+          onChangeText={(text) => setUsername(text)}
+        />
+        <TextInput
+          style={styles.inputTextBox}
+          mode="outlined"
+          label="Password"
+          value={password}
+          onChangeText={(text) => setPassword(text)}
+        />
+        <Button
+          textColor={"#000"}
```

Comment:
#### Non-blocking comment
Let's use from colors file
I am assuming eventually this button would be taken out as common component right?

Reply:
Yes, that's right. I have fixed it for my upcoming PR (for navigation+footer). But will update this PR to use colors as well.

Reply:
Done

Reply:
Ty!